alembic/versions/0019_add_entity_counters.py

```diff
@@ -0,0 +1,40 @@
+"""add entity counters
+
+Revision ID: 0019
+Revises: 0018
+Create Date: 2023-10-02 11:32:43.825217
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import Session
+
+from app.db.models.app.counters import ORGANISATION_CCLW, ORGANISATION_UNFCCC, EntityCounter
```

Comment:
where are these imports used?

Reply:
Ha I removed the code - let me fix that up - thanks :+1: